src/api/dirmonitor.c

```diff
@@ -95,8 +99,20 @@ static int f_dirmonitor_unwatch(lua_State *L) {
 }
 
 
+static int f_noop(lua_State *L) { return 0; }
```

Comment:
I wonder if we should do `core.error(...)` by default.
But given that it could raise errors, maybe just an `fprintf` is enough?